go.mod

```diff
@@ -32,7 +32,7 @@ require (
 	github.com/ipfs/go-ipfs-blockstore v0.1.4
 	github.com/ipfs/go-ipfs-chunker v0.0.5
 	github.com/ipfs/go-ipfs-cmds v0.6.0
-	github.com/ipfs/go-ipfs-config v0.12.0
+	github.com/ipfs/go-ipfs-config v0.16.1-0.20210830194616-e846165ea1ca
```

Comment:
This will need to be undone if https://github.com/ipfs/go-ipfs-config/pull/144 is merged.

Reply:
Will likely never merge this (at least not as an embedded process) so neither that PR.